deploy/lambda-env.json

```diff
@@ -39,5 +39,6 @@
   "ROLLBAR_ACCESS_TOKEN": "set_this_in_production",
   "ROLLBAR_ENDPOINT": "https://api.rollbar.com/api/1/item/",
   "DST_REFERENCE_TIMEZONE": "America/New_York",
-  "TZ": "America/New_York"
+  "TZ": "America/New_York",
+  "PHONE_NUMBER_COUNTRY": "US"
```

Comment:
Should we be careful about defaulting this to US after what Max said?

Reply:
I think the concern was about changing the behavior when unset, versus a concern about the template? FWIW the other templates all to set it--this was just updating the lambda template to match.

Reply:
I'll write a doc outlining the handful of changes you'd want to make for supporting non-US numbers soon.  [Few ideas are here](https://github.com/MoveOnOrg/Spoke/issues/1754#issuecomment-803264867) if you think of others @mayefsky!